core/src/main/java/org/apache/carbondata/core/scan/result/vector/impl/directread/ColumnarVectorWrapperDirectWithDeleteDelta.java

```diff
@@ -242,4 +242,8 @@ public void putArray(int rowId, int offset, int length) {
       columnVector.putArray(counter++, offset, length);
     }
   }
+
+  public CarbonColumnVector getColumnVector() {
```

Comment:
`columnVector` is a field defined in parent class. shall we keep this getter in its parent class(`AbstractCarbonColumnarVector`) itself.

Reply:
Right, I have made the changes.